src/utils/winner.js

```diff
@@ -0,0 +1,10 @@
+const winner = (RACE) => {
+  const RACE_ARRAY = Object.entries(RACE);
+  const maxValue = Math.max(...RACE_ARRAY.map(([key, value]) => value));
```

Comment:
```suggestion
  const maxValue = Math.max(...RACE_ARRAY.map(([key, value]) => value));
```
저는 이 부분이 `sort`, `Math.max`, `반복문` 중에 어떤거 사용 할지 굉장히 고민되었던 것 같네요🤔...